Core/HLE/sceNp.h

```diff
@@ -17,6 +17,17 @@
 
 #pragma once
 
+#include <Core/HLE/sceRtc.h>
+#include "Common/Net/HTTPClient.h"
+#include "Common/Net/Resolve.h"
+#include "Common/Net/URL.h"
+
+#ifdef _MSC_VER
+#define PACK  // on MSVC we use #pragma pack() instead so let's kill this.
+#else
+#define PACK __attribute__((packed))
+#endif
+
 #ifdef _MSC_VER
 #pragma pack(push,1)
 #endif
```

Comment:
FWIW, now in other places we simply use `#pragma pack(push, 1)` on all compilers.  See Core/Replay.cpp or GPU/Debugger/RecordFormat.h.  We don't really support ancient GCC versions anyway, so everything we compile on supports `#pragma pack(push, 1)`.

It'd be nice if there was some fancy c++25 way of doing this or something.

But anyway, this would allow you to avoid the ugly C-style struct typedefs.

-[Unknown]

Reply:
i see, i thought `#pragma pack(push, 1)` only worked for MSVC.